source/clear-linux/guides/network/dpdk.rst

```diff
@@ -1,337 +1,347 @@
    and those NICs cannot be bound.
 
    .. code-block:: bash
 
-      # dpdk-devbind --status
+      sudo dpdk-devbind --status
 
-#. Bind two available NICs. The general syntax for binding is
-   **dpdk-devbind --bind=vfio-pci <device-entry>**,
-   and the following is a working example:
+#. Bind two available NICs. The general syntax for binding is:
+   :command:`dpdk-devbind --bind=vfio-pci <device-entry>`.
+   A working example is shown below:
 
    .. code-block:: bash
 
-      # dpdk-devbind --bind=vfio-pci 01:00.0
+      sudo dpdk-devbind --bind=vfio-pci 01:00.0
 
-#. Check that your NICs binded correctly by checking the status; ``drv`` should
-   have ``igb_uio`` value; at this point, the NICs are using the DPDK modules.
+#. Check the NIC status to verify that the NICs are bound correctly. If
+   successful, `drv` displays the value `igb_uio`, which confirms
+   that the NICs are using the DPDK modules.
 
 
-Setting hugepages (platforms A and B)
-=====================================
+Set hugepages (Platforms A and B)
+*********************************
 
-Clear Linux OS for Intel Architecture supports ``hugepages`` for the large
-memory pool allocation used for packet buffers.
+|CLOSIA| supports `hugepages` for the large memory pool allocation used for
+packet buffers.
 
-#. Set number of hugepages.
+#. Set the number of hugepages.
 
    .. code-block:: bash
 
-      # echo 1024 > /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
+      sudo echo 1024 > /sys/kernel/mm/hugepages/hugepages-2048kB/nr_hugepages
 
 #. Allocate pages on NUMA machines.
 
    .. code-block:: bash
 
-      # echo 1024 > /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages
-      # echo 1024 > /sys/devices/system/node/node1/hugepages/hugepages-2048kB/nr_hugepages
+      sudo echo 1024 > /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages
+      sudo echo 1024 > /sys/devices/system/node/node1/hugepages/hugepages-2048kB/nr_hugepages
 
 #. Make memory available for DPDK.
 
    .. code-block:: bash
 
-      # mkdir -p /mnt/huge $ mount -t hugetlbfs nodev /mnt/huge
+      sudo mkdir -p /mnt/huge $ mount -t hugetlbfs nodev /mnt/huge
 
-   If you would like to know more about this, refer to the `DPDK guide`_.
+   For more information, refer to the `DPDK guide`_ System Requirements
+   section.
 
 
-Setting a physical environment (Platforms A and B)
-==================================================
+Set up the physical environment (Platforms A and B)
+***************************************************
 
-To achieve the model proposed in the introduction of this topic, (:ref:`f1`),
-we need to connect the first Grantley’s NICs to the second Grantley’s NICs
-using the network cables, see :ref:`figure 2<f2>`.
+Connect the NICs on Platform A to the NICs on Platform B using the network
+cables as shown in :ref:`Figure 2<f2>`.
 
 .. _f2:
 
 .. figure:: ./figures/pyshical_net.png
```

Comment:
"physical" is spelled wrong in the .png filename and the reference shown here. Should this be corrected?